### DISCUSSION TITLE
Can't get access to Wazuh Google Groups anymore

### DISCUSSION THREAD

**Discussion by EHRETic** [wazuh/wazuh#20666] [Q&A] [answered]
Hi there,

It seems there is a problem with Wazuh's google groups, I can't access anymore.
Don't know if it my user or if it a general problem with the group itself.
I get something like :

_Excluded content warning
Wazuh | Mailing List has been reported to contain spam, or malicious software or content.
To learn more about the Google Groups content policy, see this Help Center article on misuse and our Terms of Use.
_

![image](https://github.com/wazuh/wazuh/assets/28634810/1be3519c-4bb3-46cc-8cdc-dff66668c76e)

Does anybody else has the problem?

Kind regards
Franck

**Comment by bonartg** [comment]
Yes, I have the same problem. Does anyone from Wazuh community has an explanation regarding this issue?

**Reply by suricatalfon** [reply]
Hí,

It remains the same and there is no information about what is happening.

**Comment by rauldpm** [comment, marked as answer]
Hello @EHRETic @bonartg @suricatalfon 

We are experiencing a temporary disruption in Google Groups Mailing List Service

We acknowledge the current issues affecting the Google Groups Mailing List and want to assure you that our team is actively working on a solution. Rest assured, we will notify you once the matter is resolved.

Meanwhile, we encourage you to utilize our alternative community channels for any inquiries. You can engage with our community in Slack, Discord, and Reddit.

We appreciate your patience and understanding during this period.

Thank you in advance for your cooperation.

Wazuh Team.

**Reply by EHRETic** [reply]
Thanks for the reply, hopefully, we'll get some solution form Google soon 😊👌

**Reply by rauldpm** [reply]
Hello @EHRETic, Google Groups is accessible again! If you have any problems, don't hesitate to let us know!

**Reply by EHRETic** [reply]
Hi @rauldpm 

It works fine again, thanks a lot! 😉👌

**Reply by bonartg** [reply]
Awesome! Thanks for the update and assistance 👌
